### DISCUSSION TITLE
Configure jellyfin-vue as default / additional client for Jellyfin server

### DISCUSSION THREAD

**Discussion by alvitali** [jellyfin/jellyfin-vue#2448] [Questions] [answered]
I really like the Vue client, and would love to make it available to my end users (non-admins) as a way to stream content without any distractions.

However, the non-technically inclined crowd will not be able to / want to input the server address when loading jellyfin-vue. Is there any way to permanently point the vue-client to a Jellyfin install on the same machine? Like with env variables?

If not, I'd love to see this option! As of right now, there's no real use case for the Vue client imo.

**Comment by ferferga** [comment, marked as answer]
See https://github.com/jellyfin/jellyfin-vue/wiki/Configuration

As mentioned [here](https://github.com/jellyfin/jellyfin-vue/discussions/2446#discussioncomment-10565799), I'm trying to get the deployment docs finished this week, which will be the entrypoint for all the wiki pages.

**Reply by alvitali** [reply]
Thank you so much, this is amazing! I have no clue how I didn't manage to find the Configuration docs up until now... never realized I could already set the default server! :D

I love to see better documentation for jellyfin-vue, I'd especially welcome pointers for docker compose as well (including e.g. the env variables I hadn't found without your help). Thank you for your work on the client <3

**Comment by ferferga** [comment]
@alvitali I created a PR at the Jellyfin documentation here: https://github.com/jellyfin/jellyfin.org/pull/1110

If you don't want to read it from the source code, you can check the rendered version from the link of jellyfin-bot's comment.
Feedback at that PR is also welcome in case something there can be improved!

Although the PR is not merged yet, it will soon, so I'll close this discussion.

**Reply by alvitali** [reply]
Thanks a lot, PR is great! This will definitely help others too. I'll go through it in detail and try to fix the issues I'm having. I suspect it's something to do with my reverse proxy / nginx running in the container. Cheers
